snapshots/animal_welfare/2023-08-08/farmed_finfishes_used_for_food.zip.dvc

```diff
@@ -0,0 +1,26 @@
+meta:
```

Comment:
For some reason the metadata has not passed through to my local grapher, probably a me problem but thought I would flag it anyway.

Reply:
Hmm I do see the metadata (the old metadata fields still, if that's what's puzzling you).

Reply:
Ah yes I see the same thing. I'm looking only at metadata at the variable level, I don't really trust the admin (which is going to change very soon).